### PR TITLE
Helm updated

### PR DESCRIPTION
**Pull request by firemanxbr**
* Helm updated to 2.8.1 based in latest client [1] and Docker image [2]
* Updated the documentation to run helm command without extra parameters like --host or --kube-context

[1] https://github.com/kubernetes/helm/releases
[2] https://hub.docker.com/r/lachlanevenson/k8s-helm/

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@firemanxbr I tried to test the add-on, Here are steps I did
```
$ minishift addon apply helm

$ export HELM_HOST="$(minishift ip):$(oc get svc/tiller -o jsonpath='{.spec.ports[0].nodePort}' -n kube-system --as=system:admin)"
-bash: oc: command not found
```
After I ran `eval $(minishift oc-env)` then  it was fine. So I think you should add the dependency on oc to the readme. 

However when I try to run wordpress through helm it fails. 

```
$ helm init -c

$ helm search wordpress
NAME            	CHART VERSION	APP VERSION	DESCRIPTION
stable/wordpress	0.8.8        	4.9.4      	Web publishing platform for building blogs and ...

$ helm install stable/wordpress
NAME:   wiggly-saola
LAST DEPLOYED: Tue Mar  6 12:11:13 2018
NAMESPACE: myproject
STATUS: DEPLOYED

RESOURCES:
==> v1/ConfigMap
NAME                        DATA  AGE
wiggly-saola-mariadb        1     1s
wiggly-saola-mariadb-tests  1     1s

==> v1/PersistentVolumeClaim
NAME                    STATUS  VOLUME  CAPACITY  ACCESS MODES  STORAGECLASS  AGE
wiggly-saola-mariadb    Bound   pv0021  100Gi     RWO,ROX,RWX   1s
wiggly-saola-wordpress  Bound   pv0027  100Gi     RWO,ROX,RWX   1s

==> v1/Service
NAME                    TYPE          CLUSTER-IP     EXTERNAL-IP                  PORT(S)                     AGE
wiggly-saola-mariadb    ClusterIP     172.30.84.39   <none>                       3306/TCP                    1s
wiggly-saola-wordpress  LoadBalancer  172.30.101.30  172.29.245.12,172.29.245.12  80:30034/TCP,443:32182/TCP  1s

==> v1beta1/Deployment
NAME                    DESIRED  CURRENT  UP-TO-DATE  AVAILABLE  AGE
wiggly-saola-mariadb    1        0        0           0          0s
wiggly-saola-wordpress  1        1        1           0          0s

==> v1/Pod(related)
NAME                                    READY  STATUS             RESTARTS  AGE
wiggly-saola-wordpress-149802316-jb4r5  0/1    ContainerCreating  0         0s

==> v1/Secret
NAME                    TYPE    DATA  AGE
wiggly-saola-mariadb    Opaque  2     1s
wiggly-saola-wordpress  Opaque  2     1s


NOTES:
1. Get the WordPress URL:

  NOTE: It may take a few minutes for the LoadBalancer IP to be available.
        Watch the status with: 'kubectl get svc --namespace myproject -w wiggly-saola-wordpress'

  export SERVICE_IP=$(kubectl get svc --namespace myproject wiggly-saola-wordpress -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
  echo http://$SERVICE_IP/admin

2. Login with the following credentials to see your blog

  echo Username: user
  echo Password: $(kubectl get secret --namespace myproject wiggly-saola-wordpress -o jsonpath="{.data.wordpress-password}" | base64 --decode)

```
But it is failing 
```
$ oc get pods
NAME                                     READY     STATUS              RESTARTS   AGE
wiggly-saola-wordpress-149802316-jb4r5   0/1       ContainerCreating   0          24s
```
Does it work for you?

**Comment by LalatenduMohanty** [comment]
I guess I did not do proper steps. Will try it gain. Found https://blog.openshift.com/deploy-helm-charts-minishifts-openshift-local-development/

**Comment by firemanxbr** [comment]
@LalatenduMohanty Thank you for this feedback. I'm totally agree to improve the README file inserting more steps about the ~/.bashrc and what we expect before to start adding addons of Minishift. 
I'm reviewing the README and in the next commit I'll send it.

**NOTE:** My full test here: https://gist.github.com/firemanxbr/66925231717530a6b20fcfb6c1f0b0e4

**Comment by firemanxbr** [comment]
**ERROR**
After start the wordpress I found this errors:

$ oc get pods
NAME                                       READY     STATUS             RESTARTS   AGE
killjoy-stoat-wordpress-3092961266-fszgg   0/1       CrashLoopBackOff   6          10m

**Logs into this POD:**
Welcome to the Bitnami wordpress container
Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-wordpress 
Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-wordpress/issues 
Send us your feedback at containers@bitnami.com

WARN  ==> You set the environment variable ALLOW_EMPTY_PASSWORD=yes. For safety reasons, do not use this flag in a production environment.
INFO  ==> Starting wordpress... 
nami    ERROR Cannot find any module matching the provided specification 'apache'

In that case this error is about this specific deployment and not about helm. I hope it.

**Comment by firemanxbr** [comment]
@LalatenduMohanty I added the 'version' file to generated simply verification about the client and tiller server compatibility and improve the documentation reviewing all instructions into README.

File: add-ons/helm/README.md [hunk 1]
```diff
@@ -1,41 +1,69 @@
...
+```
+
+
 ## Discover tiller host
-Get Tiller host URL by running these commands in the shell:
+Get `oc`, `helm_host`(tiller) and `minishift` by adding this new variable into ~/.bashrc
 
 ```
-$ export TILLER_HOST="${minishift ip}:$(oc get svc/tiller -o jsonpath='{.spec.ports[0].nodePort}' -n kube-system --as=system:admin)"
+export PATH="~/.minishift/cache/oc/v3.7.1/linux:$PATH"
```
Comment (LalatenduMohanty) [inline add-ons/helm/README.md]: I would suggest replace the above with `eval $(minishift oc-env)`. The above has a hard coded version and that's something we should avoid.

**Comment by LalatenduMohanty** [comment]
@firemanxbr Please take a look at https://github.com/minishift/minishift-addons/blob/master/CONTRIBUTING.adoc#contributing-patches and update accordingly. I think you are fixing #65, right?

**Comment by firemanxbr** [comment]
@LalatenduMohanty thanks for your review. I fixed the suggestion in the README.md and helm.addon too. 
Created the right format to commit message and pushed only a commit following the squash strategy. 
Can you validate again?

**Pull request merged**